src/components/todo/TodoItem.tsx

```diff
@@ -3,58 +3,141 @@ import { TodoModal } from "../../models";
 import { dateConverter } from "../../utils/helper";
 import { useDispatch } from "react-redux";
 import { useState } from "react";
-import { deleteTodo, markDoneTodo } from "../../redux";
+import { deleteTodo, markDoneTodo, editTodo } from "../../store";
```

Comment:
We discussed this yesterday, to give is some meaningful prefix or suffix.
It doesn't give any idea what these are, are they reducer/action/actionType/state/constant.

Reply:
Updated